src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1,14 +1,15 @@
 {
   "tables": [
     {
-      "tableName": "invoice",
+      "tableName": "invoices",
```

Comment:
I think we've been inconsistent with this across the acquisitions modules... I tend to agree that the table name should be plural, maybe I/we should take account of where we stand with this and try to bring some consistency before go-live.  After that it would be increasingly difficult to make these sorts of changes as customer data might need to be migrated from one table to another

Reply:
I agree with you, Craig. Maybe we can document this on wiki page as a rule or best practice for future? And, probably, we can inspect now already existed tables for renaming them?